docs/limits.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 4
+sidebar_class_name: sidebar-item-limits
+---
+
+# Limits 
+
+We have several default limits in place at Momento to help protect you as you get up and running. This page will outline our default service limits that you should be aware of as you look at using Momento as a caching provider. Please note that all the limits can be adjusted up as needed, we can work with you to adjust to support your workloads. Please reach out to hello@momentohq.com to increase limits if default are to low. (Self service coming soon!)
+
+| Limit             | Value         |
+|-------------------|---------------|
+| Item Size         | 1MB           |
+| Max Cache Count   | 10            |
+| Data Plane TPS    | 100 TPS/Cache |
+| Control Plane TPS | 20 TPS        |
```

Comment:
Define "TPS" before this

Reply:
ack

Reply:
i thought i remembered a convo somewhere where we had decided to go with "RPS" instead of "TPS" to avoid confusion with rdbms transactions?

Reply:
updated!